<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-QPPOL30" failures="1" tests="5" name="tests.LoginTests" time="15.261" errors="0"
           timestamp="2020-05-02T11:51:47 PDT" skipped="1">
    <testcase classname="tests.LoginTests" name="test_Sum" time="3.633">
        <failure message="expected [7] but found [6]" type="java.lang.AssertionError">
            <![CDATA[java.lang.AssertionError: expected [7] but found [6]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at tests.LoginTests.test_Sum(LoginTests.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
        </failure>
    </testcase> <!-- test_Sum -->
    <system-out/>
    <testcase classname="tests.LoginTests" name="test_Sum" time="3.581"/>
    <system-out/>
    <testcase classname="tests.LoginTests" name="test_Sum" time="3.586"/>
    <system-out/>
    <testcase classname="tests.LoginTests" name="test_Sum" time="4.461"/>
    <system-out/>
    <testcase classname="tests.LoginTests" name="test_Fails">
        <skipped/>
    </testcase> <!-- test_Fails -->
    <system-out/>
</testsuite> <!-- tests.LoginTests -->
